internal/pkg/plugin/trello/create.go

```diff
@@ -0,0 +1,28 @@
+package trello
+
+import (
+	"github.com/merico-dev/stream/pkg/util/log"
+)
+
+// Create sets up trello.
```

Comment:
"sets up" isn't quite self-explanatory. It's better to put the exact behavior, such as, "it creates a Trello board and some columns", etc.